scripts/split-mems-conf.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
```

Comment:
Could we use `#!/usr/bin/env python3`? On my mac, the `python` binary either doesn't exist, or actually points to python 2. Are any Linux systems _only_ shipping with `python` binaries which are actually python 3?